contracts/strategies/idle/IdleFinanceStrategy.sol

```diff
@@ -171,49 +172,61 @@ contract IdleFinanceStrategy is IStrategy, RewardTokenProfitNotifier {
-      // we can accept 1 as the minimum because this will be called only by a trusted worker
-      IUniswapV2Router02(uniswapRouterV2).swapExactTokensForTokens(
-        compBalance, 1, uniswapComp, address(this), block.timestamp
-      );
+    uint256 claimableAave = IStakedAave(stkaave).stakerRewardsToClaim(address(this));
+    if (claimableAave > 0) {
+      IStakedAave(stkaave).claimRewards(address(this), claimableAave);
     }
   }
 
-  function liquidateIdle() internal {
-    if (!sellIdle) {
-      // Profits can be disabled for possible simplified and rapid exit
-      emit ProfitsNotCollected(idle);
-      return;
+  function liquidateRewards() internal {
+    for (uint256 i=0;i<rewardTokens.length;i++) {
+      address token = rewardTokens[i];
+      if (!sell[token]) {
+        // Profits can be disabled for possible simplified and rapid exit
+        emit ProfitsNotCollected(token);
+        continue;
+      }
+
+      uint256 balance = IERC20(token).balanceOf(address(this));
+      if (balance > 0) {
+        emit Liquidating(token, balance);
+        address routerV2;
+        if(useUni[token]) {
+          routerV2 = uniswapRouterV2;
+        } else {
+          routerV2 = sushiswapRouterV2;
+        }
+        IERC20(token).safeApprove(routerV2, 0);
+        IERC20(token).safeApprove(routerV2, balance);
+        // we can accept 1 as the minimum because this will be called only by a trusted worker
+        IUniswapV2Router02(routerV2).swapExactTokensForTokens(
+          balance, 1, reward2WETH[token], address(this), block.timestamp
+        );
+      }
     }
 
-    uint256 rewardBalance = IERC20(idle).balanceOf(address(this));
-    notifyProfitInRewardToken(rewardBalance);
-
-    uint256 idleBalance = IERC20(idle).balanceOf(address(this));
-    if (idleBalance > 0) {
-      emit Liquidating(address(idle), idleBalance);
-      IERC20(idle).safeApprove(uniswapRouterV2, 0);
-      IERC20(idle).safeApprove(uniswapRouterV2, idleBalance);
+    uint256 wethBalance = IERC20(weth).balanceOf(address(this));
+    notifyProfitInRewardToken(wethBalance);
+
+    uint256 remainingWethBalance = IERC20(weth).balanceOf(address(this));
+
+    if (remainingWethBalance > 0) {
+      emit Liquidating(weth, remainingWethBalance);
+      address routerV2;
+      if(useUni[address(underlying)]) {
+        routerV2 = uniswapRouterV2;
+      } else {
+        routerV2 = sushiswapRouterV2;
+      }
+      IERC20(weth).safeApprove(routerV2, 0);
+      IERC20(weth).safeApprove(routerV2, remainingWethBalance);
       // we can accept 1 as the minimum because this will be called only by a trusted worker
-      IUniswapV2Router02(uniswapRouterV2).swapExactTokensForTokens(
-        idleBalance, 1, uniswapIdle, address(this), block.timestamp
+      IUniswapV2Router02(routerV2).swapExactTokensForTokens(
+        remainingWethBalance, 1, WETH2underlying, address(this), block.timestamp
       );
     }
+    uint256 balance = underlying.balanceOf(address(this));
```

Comment:
Is it used?

Reply:
No, used it to log in testing, forgot to remove